shared/markdown/parser.pegjs

```diff
@@ -117,14 +117,20 @@ __INLINE_RULE__<StrikeInline, !StrikeMarker>
 Strike
  = StrikeMarker !WhiteSpace children:StrikeInline StrikeMarker !(StrikeMarker / NormalChar) { return {type: 'strike', children: flatten(children)} }
 
-// children grammar adapted from username regexp in libkb/checkers.go.
+// children test adapted from CheckUsername in libkb/checkers.go.
 Mention
- = MentionMarker children:([a-zA-Z0-9][a-zA-Z0-9_]*) MentionMarker service:ValidMentionService { return {type: 'mention', children: flatten(children), service: service.toLowerCase()} }
+ = MentionMarker children:([a-zA-Z0-9]+[a-zA-Z0-9_]?)+ MentionMarker service:ValidMentionService & {
```

Comment:
could `[a-zA-Z0-9_]?` -> `[_]?` here? if I'm understanding how the grammar works, that would match `[a-zA-Z0-9]+` repeatedly and allow a single underscore in between

Reply:
Yes! I don't know why I didn't have that. Will change on Monday...